mypy.ini

```diff
@@ -0,0 +1,26 @@
+[mypy]
+python_version = 3.6
+files = src/sentry/snuba/query_subscription_consumer.py
+
+; Enable all options used with --strict
+warn_unused_configs=True
+disallow_any_generics=True
+disallow_subclassing_any=True
+disallow_untyped_calls=True
+disallow_untyped_defs=True
+disallow_incomplete_defs=True
+check_untyped_defs=True
+disallow_untyped_decorators=True
+no_implicit_optional=True
+warn_redundant_casts=True
+warn_unused_ignores=True
+warn_return_any=True
+no_implicit_reexport=True
+
+
+[mypy-confluent_kafka.*]
+ignore_missing_imports = True
+[mypy-jsonschema]
+ignore_missing_imports = True
+[mypy-django.*]
+ignore_missing_imports = True
```

Comment:
We might be able to get type stubs for some of these, but not trying to just yet.